civihr_employee_portal/views/templates/views-view-table--Tasks--block.tpl.php

```diff
@@ -65,6 +65,7 @@
 
 function _get_task_filter_by_date($date) {
     $today = date('Y-m-d');
+    $tomorrow = new DateTime('tomorrow');
```

Comment:
would it be hard to refactor to only use DateTime objects for date/time here, instead of strings?